Dockerfile

```diff
@@ -45,6 +45,7 @@ ENV SESSION_WARN=1500
 ENV SESSION_EXPIRE=1800
 ENV STATIC_ROOT="${TETHYS_PERSIST}/static"
 ENV WORKSPACE_ROOT="${TETHYS_PERSIST}/workspaces"
+ENV MEDIA_ROOT="${WORKSPACE_ROOT}"
```

Comment:
```suggestion
ENV MEDIA_ROOT="${WORKSPACE_ROOT}"
ENV MEDIA_URL="/media/"
```

Reply:
Need to create this directory somewhere in the Dockerfile.

Reply:
Changed it to {TETHYS_PERSIST}/media" and added it to the MKDIR command